apps/price_pusher/src/solana/solana.ts

```diff
@@ -174,7 +174,7 @@ export class SolanaPricePusherJito implements IPricePusher {
   async getRecentJitoTipLamports(): Promise<number | undefined> {
     try {
       const response = await fetch(
-        "http://bundles-api-rest.jito.wtf/api/v1/bundles/tip_floor",
+        "https://bundles.jito.wtf/api/v1/bundles/tip_floor",
       );
       if (!response.ok) {
         this.logger.error(
```

Comment:
shall we make it warning? because if it doesn't work we default to something else (and hopefully other alerts should catch it up if it results into not landing txs)

Reply:
yup good point will make the change